SD-K9/src/app/models/view-mode.enum.model.ts

```diff
@@ -0,0 +1,6 @@
+export enum ViewMode {
+    GOOGLE = "GOOGLE",
+    HALL = "HALL",
+    LOYOLA = "LOYOLA",
```

Comment:
Maybe this should be 
```suggestion
    CC = "CC",
```